src/Migration/Run/MigrationStep.php

```diff
@@ -16,6 +16,8 @@ enum MigrationStep: string
 
     case FETCHING = 'fetching';
 
+    case APPLY_FIXES = 'apply-fixes';
```

Comment:
Would be great to adjust these doc comments to include the new step:
https://github.com/shopware/SwagMigrationAssistant/blob/5a9afd81133508891430770194d4ee94d41ce6cc/src/Migration/Run/RunServiceInterface.php#L17-L35

Reply:
Nevermind, you already did that 👍